components/Login.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+import Image from 'next/image';
+import { login } from '../app/(auth)/login/actions';
+import Logo from '@/public/images/logo-placeholder-image.png';
+import Link from 'next/link';
+import { LoginSchema } from '@/lib/schemas';
+import { useToast } from './ui/use-toast';
+
+const Login = () => {
+  const { toast } = useToast();
+  const validateAndLoginUser = (formData: FormData) => {
+    const userData = {
+      email: formData.get('email') as string,
+      password: formData.get('password') as string,
+    };
+
+    const result = LoginSchema.safeParse(userData);
+
+    if (result.success === false) {
```

Comment:
```suggestion

    if (!result.success) {
```

Reply:
I prefer "=== false" - for me it's easier to find it in code, but I can change it if you want